.github/workflows/industrial_ci_action.yaml

```diff
@@ -0,0 +1,70 @@
+# This config uses industrial_ci (https://github.com/ros-industrial/industrial_ci.git).
+# For troubleshooting, see readme (https://github.com/ros-industrial/industrial_ci/blob/master/README.rst)
+
+name: BuildAndTest
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - noetic-devel
```

Comment:
This only applies to the melodic-devel branch
```suggestion
```

Reply:
done, will do